static/app/views/replays/detail/tagPanel/index.tsx

```diff
@@ -68,25 +70,34 @@ function TagPanel() {
     <FluidHeight>
       <TagFilters tags={tags} {...filterProps} />
       <TabItemContainer>
-        <FluidPanel>
-          {filteredTags.length ? (
-            <KeyValueTable noMargin>
-              {filteredTags.map(([key, values]) => (
-                <ReplayTagsTableRow
-                  key={key}
-                  name={key}
-                  values={values}
-                  generateUrl={generateUrl}
-                />
-              ))}
-            </KeyValueTable>
-          ) : (
-            <EmptyMessage>{t('No tags for this replay were found.')}</EmptyMessage>
-          )}
-        </FluidPanel>
+        <StyledPanel>
+          <FluidPanel>
+            {filteredTags.length ? (
+              <KeyValueTable noMargin>
+                {filteredTags.map(([key, values]) => (
+                  <ReplayTagsTableRow
+                    key={key}
+                    name={key}
+                    values={values}
+                    generateUrl={generateUrl}
+                  />
+                ))}
+              </KeyValueTable>
+            ) : (
+              <EmptyMessage>{t('No tags for this replay were found.')}</EmptyMessage>
+            )}
+          </FluidPanel>
+        </StyledPanel>
       </TabItemContainer>
     </FluidHeight>
   );
 }
 
+const StyledPanel = styled(Panel)`
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  margin-bottom: 0;
+`;
+
```

Comment:
There's a double-border because Panel is the base element here.

I think we could get away with only using a few styles:
```suggestion
const StyledPanel = styled('div')`
  position: relative;
  height: 100%;
  overflow: auto;
  display: grid;
`;

```